{
    "affected": [
        {
            "package": {
                "ecosystem": "Debian:8",
                "name": "python-django",
                "purl": "pkg:deb/debian/python-django?distro=jessie&repository_url=http%3A%2F%2Fdeb.freexian.com%2Fextended-lts"
            },
            "ranges": [
                {
                    "events": [
                        {
                            "introduced": "0"
                        },
                        {
                            "fixed": "1.7.11-1+deb8u20"
                        }
                    ],
                    "type": "ECOSYSTEM"
                }
            ]
        }
    ],
    "database_specific": {
        "type": "security update"
    },
    "id": "ELA-1448-1",
    "modified": "2025-05-31T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://deb.freexian.com/extended-lts/tracker/ELA-1448-1"
        },
        {
            "type": "ADVISORY",
            "url": "https://deb.freexian.com/extended-lts/tracker/CVE-2023-36053"
        },
        {
            "type": "ADVISORY",
            "url": "https://deb.freexian.com/extended-lts/tracker/CVE-2024-24680"
        },
        {
            "type": "ADVISORY",
            "url": "https://deb.freexian.com/extended-lts/tracker/CVE-2025-32873"
        }
    ],
    "related": [
        "CVE-2023-36053",
        "CVE-2024-24680",
        "CVE-2025-32873"
    ]
}